features/tests_by_category_metrics.feature

```diff
@@ -0,0 +1,290 @@
+@api @metrics
+Feature: Tests by categories
+
+  Users should be able to retrieve the number of tests by categories for latest project versions. When a project version
+  is not provided, we gather the data from the latest project versions based on creation date.
+
+  The following filters are available:
+  - no filter (in fact, organization filter is applied and is mandatory for every filter)
+  - project version
+  - project
+  - user
+  - project and user
+  - project version and user
+
+  The following result is provided:
+  - name of the category
+  - the number of tests for this category
+
+  In some cases, tests do not have a category. In this case, a dedicated property is available to get
```

Comment:
This description lies. The `noCategoryTestsCount` property is not available only "in this case". It is always returned by the API, whether there are tests with no category or not.